OpenKeychain/src/main/java/org/sufficientlysecure/keychain/ui/EncryptModeAsymmetricFragment.java

```diff
@@ -50,6 +52,7 @@
 
     private KeySpinner mSignKeySpinner;
     private EncryptKeyCompletionView mEncryptKeyView;
+    private String mQuery;
```

Comment:
Why is this global? Try to get rid of all the warnings shown in Android Studio, that should prevent issues like this ("field can be converted to local").